static/js/components/dashboard/CourseStatus.js

```diff
@@ -1,50 +1,28 @@
 // @flow
```

Comment:
more work? we need _some_ component in the middle there. its name is likely to change, but i'd much rather leave the placeholder than delete it and worry about cleaning up unit tests